doc/api/http.md

```diff
@@ -74,9 +74,9 @@ to keep the Node.js process running when there are no outstanding requests.
 It is good practice, to [`destroy()`][] an `Agent` instance when it is no
 longer in use, because unused sockets consume OS resources.
 
-Sockets are removed from an agent's pool when the socket emits either
+Sockets are removed from an agent when the socket emits either
```

Comment:
Agent's pool is `agent.freeSockets`. When a socket is assigned to a request, the socket must not be in `agent.freeSockets`.